digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectDeltaCalculator.scala

```diff
@@ -217,28 +217,24 @@ object ProjectDeltaCalculator {
       (matcherStartMAddress < targetStartMAddress && matcherEndMAddress > targetStartMAddress) || (matcherStartMAddress < targetEndMAddress && matcherEndMAddress > targetEndMAddress)
   }
 
-  def partition[T <: BaseRoadAddress](roadAddresses: Seq[ProjectLink]): Seq[RoadwaySection] = {
-    val grouped = roadAddresses.groupBy(ra => (ra.roadNumber, ra.roadPartNumber, ra.track, ra.roadType))
+  def partition[T <: BaseRoadAddress](projectLinks: Seq[ProjectLink]): Seq[RoadwaySection] = {
+    val grouped = projectLinks.groupBy(ra => (ra.roadNumber, ra.roadPartNumber, ra.track, ra.roadType))
```

Comment:
What about discontinuity? Are these changes split also on discontinuity places?

Reply:
This method groups all links. Discontinuities are handled in combineTwo method. We combine project links one-by-one, generating a "temporary" section, until it finds a discontinuity